MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1145 @@
+</td>
+<td>
+{{'[DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Time
+</td>
+<td>
+The TimeSpan and TimeOnly type property or the property with [DataType(DataType.Time)] attribute.
+</td>
+<td>
+{{'[TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Checkbox
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Switch
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+Enum and List type property. 
+</td>
+<td>
+{{'[Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[SfComboBox](https://help.syncfusion.com/maui/combobox)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[SfAutoComplete](https://help.syncfusion.com/maui/autocomplete)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+</table>
+
+## Change the editor for any data type
+
+By default, the editors will be loaded based on the primitive data type such as string, enumeration, DateTime, and TimeSpan.
+
+{% tabs %}
+{% highlight C# %}
+
+    this.dataForm.RegisterEditor(typeof(string), DataFormEditorType.MultilineText);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the `MultilineText` editor will be loaded for the string type instead of the `Text` editor.
+
+## Change the editor for property
+
+To change the editor for any property, use the `RegisterEditor` method and specify the property name and editor.
+
+{% tabs %}
+{% highlight c# %}
+
+    this.dataForm.RegisterEditor("On", DataFormEditorType.Switch);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the Switch editor will be loaded for the `On` property (bool type) instead of the `CheckBox` editor.
+
+## Text editor
+
+In the text editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Text)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+In `Text`, `MultilineText` and `Password` editors you can change the soft input keyboard type by using the `Keyboard` property of `DataFormTextEditorItem`.
```

Comment:
**Mention in all editor above example code**
GenerateDataFormItem event will be trigger to customize the data editor only for AutoGenartae true.